edb/edgeql-parser/src/ast.rs

```diff
@@ -0,0 +1,2633 @@
+// DO NOT EDIT. This file was generated with:
```

Comment:
Is it temporary, while we have not merged that to master. Or do you expect python to be source of truth for the long time?

Reply:
That's here to stay as long as we have the eql compiler written in Python. That's because we want to have Python definitions of the AST so mypy works correctly and because we want to be able to construct AST nodes from Python too.

We could generate Python definitions from Rust ones, but from my experiments, it's easier to do it this way around.